main_test.go

```diff
@@ -254,7 +254,7 @@ func TestControllerConfigmapUpdate(t *testing.T) {
 			shouldBeUpdated: true,
 		},
 		{
-			name:            "ConfigMapWithOtherChange",
+			name:            "ConfigMapWithSameHashringAndOtherChanges",
```

Comment:
what are the `other changes` in this test? Isn't the initial configmap identical, and thus `shouldBeUpdated` is false?

Reply:
The other changes are the label changes.

>  Isn't the initial configmap identical, and thus `shouldBeUpdated` is false?

Yes, exactly.

Maybe I should be more specific like `ConfigMapWithSameHashringAndLabelChanges`? I can also add test cases like `ConfigMapWithSameHashringAndAnnotationChanges`.

I just want to emphasis and elaborate update behaviour that we have in reconciliation loop. WDYT?

Reply:
Naming is hard. The only issue here is that IMO super long test case names are hard to read and parse for humans especially when they are in camel case. This test is trying to show that the original labels will be kept, right? In that case call it `KeepLabels` and call the other one `DifferentHashring`.